apps/mobile/src/components/ChatDetailScreen/components/AcceptDeclineButtons.tsx

```diff
@@ -33,7 +33,15 @@ function AcceptDeclineButtons({
         TE.match(
           (e) => {
             loadingOverlay.hide()
-            Alert.alert(toCommonErrorMessage(e, t) ?? t('common.unknownError'))
+            if (e._tag === 'RequestCancelledError') {
+              Alert.alert(t('offer.requestWasCancelledByOtherSide'))
```

Comment:
This can happen when notification is not delivered for some reason. It's good to let user know that the request s/he is trying to accept is cancelled than to display generic error